rust/connlib/tunnel/src/peer.rs

```diff
@@ -379,12 +379,12 @@ impl ClientOnGateway {
     ) -> anyhow::Result<Option<IpPacket>> {
         let (proto, ip) = match self.nat_table.translate_incoming(&packet, now)? {
             TranslateIncomingResult::Ok { proto, src } => (proto, src),
-            TranslateIncomingResult::DestinationUnreachable(prototype) => {
-                tracing::debug!(dst = %prototype.outside_dst(), proxy_ip = %prototype.inside_dst(), error = ?prototype.error(), "Destination is unreachable");
+            TranslateIncomingResult::ICMPError(prototype) => {
+                tracing::debug!(dst = %prototype.outside_dst(), proxy_ip = %prototype.inside_dst(), error = ?prototype.error(), "ICMP Error");
```

Comment:
This change here currently makes the tests fail because we grep for this log message in the test output as a form of "coverage testing". See `workflows/_rust.yml`. We need to update that part.